config/routes.rb

```diff
@@ -23,8 +25,13 @@
 
   namespace :api do
     namespace :v1 do
-      get 'protocol_subscriptions/:id', to: 'protocol_subscriptions#show'
       get 'statistics', to: 'statistics#index'
+      get 'protocol_subscriptions/:id', to: 'protocol_subscriptions#show'
+      namespace :admin do
+        resources :organization, only: [:show], param: :group
+      end
     end
   end
+
+  get '*path', to: 'admin#index'
```

Comment:
Dit betekent toch gewoon dat ieder willekeurig argument dat niet door eerdere routes is afgevangen naar de admin route loopt? Waarom is dat nodig? Dit lijkt me wat slordig. Zeg maar het liefst heb je volgens mij dat mensen helemaal niet afweten van je admin interface. Zeker omdat het gewoon achter basic auth hangt. Kunnen we het niet beperken tot '/admin/*path' oid?

Reply:
Zeg maar, ik ben net naar `vsv.io/asdf` gegaan, en dan wordt ik dus naar de /admin route geredirect, alleen dan zie ik geen login button (omdat de route niet exact matcht op "/admin"). Lijkt me allemaal niet wenselijk.